flutter-studio/src/io/flutter/FlutterStudioStartupActivity.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter;
+
+import com.intellij.openapi.actionSystem.ActionManager;
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.startup.StartupActivity;
+import io.flutter.actions.OpenAndroidModule;
+import org.jetbrains.annotations.NotNull;
+
+public class FlutterStudioStartupActivity implements StartupActivity {
+  @Override
+  public void runActivity(@NotNull Project project) {
+    replaceAction("flutter.androidstudio.open", new OpenAndroidModule());
```

Comment:
Can you add some brief in-lines docs here, to say that when in Android Studio, we replace the 'open in android studio' action with one that opens a new window instead of spawn a new IDE process?

Reply:
Will do!